### PR TITLE
Modify Helm template to support multiple etcdhost, add anti-affinity

### PR DESCRIPTION
**Pull request by TkClark**
1. Suports add multiple etcdhost.
2. `etcd.port` has modified to `etcd.default`.
3. Modifiy README.me.
4. Add default anti-affinity to deployment.yaml template, default false. turn on anti-affinity by `--set affinity.enabled=true`

### CODE REVIEW

File: chart/apisix/values.yaml [hunk 1]
```diff
@@ -188,5 +190,5 @@ nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
-
+affinity: 
+  enabled: true
```
[the remark this answers is not in the document]
Comment (TkClark) [reply chart/apisix/values.yaml]: get it

**Comment by moonming** [comment]
@TkClark please use English in the public channel

**Review by gxthrj — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -44,14 +44,15 @@ helm uninstall apisix ./chart/apisix -n default
 
 1. How to install APISIX only?
 
-The Charts will install etcd 3.4.14 by default. If you want to install Apache APISIX only, please set `etcd.enabled=false` and set `etcd.host=http://your_etcd_address:2379`.
+The Charts will install etcd 3.4.14 by default. If you want to install Apache APISIX only, please set `etcd.enabled=false` and set `etcd.host={http://your_etcd_address:2379}`.
```
Comment (tokers) [inline README.md:47]: Strange, if i don't want to install etcd, why should i still care and set the etcd host?

File: README.md [hunk 3]
```diff
@@ -44,14 +44,15 @@ helm uninstall apisix ./chart/apisix -n default
...
 
 Please use the FQDN address or the IP of the etcd.
 
 ```shell
+# if etcd export by kubernetes service need spell fully qualified name
 $ helm install apisix ./chart/apisix -n default \
     --set etcd.enabled=false \
-    --set etcd.host=http://your_etcd_address:2379
+    --set etcd.host={http://etcd_node_1:2379\,http://etcd_node_2:2379}
```
Comment (tokers) [inline README.md:55]: Better to add some descriptions about how people can pass multiple etcd hosts.

File: README.md (hunk 2 above)
Comment (gxthrj) [reply README.md:47]: We can modify the `enable` extension, its meaning is not to enable etcd, but switch configuration options.

**Review by tokers — APPROVED** [review]

**Comment by tokers** [comment]
LGTM, except some docs should be altered.

File: chart/apisix/values.yaml [hunk 4]
```diff
@@ -188,5 +190,5 @@ nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
-
+affinity:
```
Comment (Junnplus) [inline chart/apisix/values.yaml:193]: break change? how to custom affinity?

Comment (Junnplus) [reply chart/apisix/values.yaml:193]: cc @tokers

Comment (Junnplus) [reply chart/apisix/values.yaml:193]: Is anyone concerned about this issue? cc @gxthrj @tokers @membphis 

In addition, `affinity.enabled` is invalid for now, https://github.com/apache/apisix-helm-chart/pull/20/files#diff-9a91f0c4f242a1392081ea0f59c672ef054db7ddfe3a9f8fe98e6ebd9b06025aR100

I suggest we can roll back first.

Comment (Junnplus) [reply chart/apisix/values.yaml:193]: more related: https://github.com/apache/apisix-helm-chart/pull/26#discussion_r569105495

**Pull request merged**